extra/mondoo-slack-security.mql.yaml

```diff
@@ -0,0 +1,64 @@
+policies:
+  - uid: mondoo-slack-security
+    name: Slack Security Best Practices by Mondoo
+    version: 1.0.0
+    is_public: true
+    authors:
+      - name: Mondoo, Inc
+        email: hello@mondoo.com
+    docs:
+      desc: |
+        ### Overview
+
+        The Mondoo Slack Security policy ensure best-practice settings for Slack.
```

Comment:
The Mondoo Slack Security policy ensures best-practice settings for Slack workspaces.